go.mod

```diff
@@ -1,23 +1,29 @@
 module github.com/gocql/gocql
 
 require (
-	github.com/bitly/go-hostpool v0.0.0-20171023180738-a3a6125de932 // indirect
-	github.com/bmizerany/assert v0.0.0-20160611221934-b7ed37b82869 // indirect
 	github.com/golang/snappy v0.0.3
```

Comment:
so why do we still need snappy?

Also note there's already 0.0.4, but with benign changes - https://github.com/golang/snappy/compare/v0.0.3...v0.0.4

Reply:
it was for tests, removed